apps/frontend/src/app/calendarPage/page.tsx

```diff
@@ -0,0 +1,129 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+
+import { styles } from '@/components/calendarStyles';
+
+// Segédfüggvények
+const daysInWeek = ['Hétfő', 'Kedd', 'Szerda', 'Csütörtök', 'Péntek', 'Szombat', 'Vasárnap'];
+const shortDaysInWeek = ['H', 'K', 'Sz', 'Cs', 'P', 'Sz', 'V'];
+
+const monthNames = [
+  'Január',
+  'Február',
+  'Március',
+  'Április',
+  'Május',
+  'Június',
+  'Július',
+  'Augusztus',
+  'Szeptember',
+  'Október',
+  'November',
+  'December',
+];
+
+const getDaysInMonth = (month: number, year: number) => {
+  return new Date(year, month + 1, 0).getDate();
+};
+
+const getFirstDayOfMonth = (month: number, year: number) => {
+  const day = new Date(year, month, 1).getDay();
+  return day === 0 ? 7 : day;
+};
+
+// Calendar komponens
+const Calendar: React.FC = () => {
+  const [currentMonth, setCurrentMonth] = useState(7); // Augusztus
+  const [currentYear, setCurrentYear] = useState(2024); // 2024
+  const [isMobile, setIsMobile] = useState(false);
+
+  // Ellenőrizze, hogy mobilon vagyunk-e
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 600);
+    };
+
+    window.addEventListener('resize', handleResize);
+    handleResize(); // Csak egyszer hívjuk meg, amikor a komponens betöltődik
```

Comment:
this comment is a bit misleading, it is called in case of the resize event as well, so please remove it